importers/ableroad.com/generate_csv.py

```diff
@@ -10,18 +16,32 @@ def generate_csv():
 	with open(filename, 'wb') as csv_file:
 		writer = csv.writer(csv_file, delimiter=',',
                             quotechar='"', quoting=csv.QUOTE_ALL)
-		writer.writerow(['Name', 'Categories','Street Address', 'Zip Code',
-			'Neighbourhood', 'Locality', 'State',
-			'Yelp Rating', 'Num Yelp Ratings', 'Yelp Review Start', 'Distance',
-			'Ableroad Rating', 'Ableroad Num Ratings', 'Ableroad Review Text', 'Thumbnail URL', 'Details URL'])
+		extra_columns = [
+			{'title': 'Name', 'key': 'name'}, {'title': 'Categories', 'key': 'categories'},
+			{'title': 'Street Address', 'key': 'street_address'}, {'title': 'Zip Code', 'key': 'zipcode'},
+			{'title': 'Neighbourhood', 'key': 'neighbourhood'}, {'title': 'Locality', 'key': 'locality'},
+			{'title': 'State', 'key': 'state'}, {'title': 'Yelp Rating', 'key': 'yelp_rating'},
+			{'title': 'Yelp Num Ratings', 'key': 'yelp_num_ratings'}, {'title': 'Yelp Reviews', 'key': 'yelp_review_start'},
+			{'title': 'Distance', 'key': 'distance'}, {'title': 'Ableroad Rating', 'key': 'ableroad_rating'},
```

Comment:
You can completely remove it from both easy_importer.py and generate_csv.py, right?